src/Components/Generator/RequestForm.js

```diff
@@ -170,16 +170,19 @@ export default class RequestForm extends preact.Component {
                                 current={this.props.request_data['type']}
                             />
 
-                            <RecipientInput
-                                onChange={e => this.props.onChange({ recipient_address: e.target.value })}
-                                recipientAddress={this.props.request_data.recipient_address}
-                            />
-
                             <TransportMediumChooser
                                 transportMedium={this.props.request_data.transport_medium}
                                 onChange={this.props.onTransportMediumChange}
                             />
 
+                            <RecipientInput
+                                onAddressChange={e => this.props.onChange({ recipient_address: e.target.value })}
+                                onEmailChange={e => this.props.onChange({ email: e.target.value })}
```

Comment:
`Request` currently doesn't have an attribute `email`. This is why changing the email address in the input doesn't change the `mailto:`-link. Currently, we use `this.state.suggestion.email`, so whatever the email may be changed to, the chosen company's email will always be used in the link.